app/core/components/ProposalCard/index.tsx

```diff
@@ -86,75 +86,72 @@ export const ProposalCard = (props: IProps) => {
                 </div>
               </div>
               <EllipsisText text={props.description || ""} length={200} />
-              <div className="ProposalCard__skills">
-                {props.skills &&
-                  props.skills[0].name != "" &&
-                  props.skills.map((skill) => (
-                    <Chip
-                      key={skill.name}
-                      label={skill.name}
-                      sx={{ marginRight: 1, marginBottom: 1 }}
-                    />
-                  ))}
-              </div>
-              {user && show && (
-                <div className="ProposalCard__status">
-                  <hr />
-
+            </ProposalCardWrap>
+          </CardContent>
+          <CardActions sx={{display: "flex", alignItems: "baseline", flexDirection: "column", cursor: "auto"}}>
+            <ProposalCardSkills>
+              {props.skills &&
+                props.skills[0].name != "" &&
+                props.skills.map((skill) => (
+                  <Chip
+                    key={skill.name}
+                    component="a"
+                    label={skill.name}
+                    sx={{ marginRight: 1, marginBottom: 1 }}
+                    clickable
+                    href={`/projects?discipline=${skill.name}`}
+                  />
+                ))}
+            </ProposalCardSkills>
+            {user && show && (
+              <ProposalCardStatus> 
+                <div>
                   <div>
-                    <div>
-                      <span className="ProposalCard__status--display">
-                        {props.status}
-                      </span>
-                      <div className="ProposalCard__tier">
-                        <Link
-                          href="https://wizeline.atlassian.net/wiki/spaces/wiki/pages/3075342381/Innovation+Tiers"
-                          target="_blank"
-                          rel="noreferrer"
-                          onClick={stopEvent}
-                          underline="hover"
-                        >
-                          <label className="ProposalCard__head__description--tier">
-                            {props.tierName}
-                          </label>
-                        </Link>
-                        <label
-                          className="ProposalCard__head__description--tier--extra"
-                          title="Maturation framework for innovation projects"
-                        >
-                          <HelpIcon />
+                    <span className="ProposalCard__status--display">
+                      {props.status}
+                    </span>
+                    <div className="ProposalCard__tier">
+                      <Link
```

Comment:
I checked and the cursor didn’t change to the correct cursor, la manita, can you change it?

Reply:
I removed the label and add the cursor as pointer